vmdb/lib/tasks/test_metrics.rake

```diff
@@ -4,7 +4,7 @@
 #   original here: https://github.com/ManageIQ/metric_fu/blob/master/tasks/metric_fu.rake
 require_relative "./evm_test_helper"
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
```

Comment:
I see how it is less duplication. but I'd prefer to just use the standard rake task loader